### PR TITLE
Add SDD

### PR DESCRIPTION
**Pull request by GoodluckH**

### CODE REVIEW

File: docs/sdd.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+<img src="https://github.com/Arrow-air/tf-github/raw/main/src/templates/doc-banner-services.png" style="height:250px" />
+
+</center>
+
+### Metadata
+
+| Item | Description |
+| --- | --- |
+| Maintainer(s) | [GoodluckH](https://github.com/GoodluckH) |
```
Comment (owlot) [inline docs/sdd.md]: I think we just want to add the [services team](https://github.com/orgs/Arrow-air/teams/services) as Maintainers for the libraries / services.
@ServiceDog what do you think?

Comment (amsmith-pro) [reply docs/sdd.md]: Yes please, though I do also see the value of adding a "Primary Author" or "Stuckee" field

Comment (GoodluckH) [reply docs/sdd.md]: [32376a1](https://github.com/Arrow-air/lib-router/pull/10/commits/32376a1818218a0472f03df890e34d76ed744650)

**Review by amsmith-pro — APPROVED** [review]
Nice work

**Pull request merged**